spec/features/workflow/change_scheduled_publishing_datetime_spec.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+RSpec.feature "Change scheduled publishing datetime" do
+  scenario do
+    given_there_is_an_edition_with_a_scheduled_publishing_datetime
+    when_i_visit_the_summary_page
+    and_i_click_on_change_date
+    then_i_see_the_current_date_and_time_selected
+
+    when_i_choose_a_new_scheduled_date_and_time
+    and_i_click_save_date
+    then_i_see_the_new_scheduled_date_and_time
+  end
+
+  def given_there_is_an_edition_with_a_scheduled_publishing_datetime
+    @current_datetime = Time.current.tomorrow.change(hour: 10)
+    @edition = create(:edition, scheduled_publishing_datetime: @current_datetime)
+  end
+
+  def when_i_visit_the_summary_page
+    visit document_path(@edition.document)
+  end
+
+  def and_i_click_on_change_date
+    click_on "Change date"
+  end
+
+  def then_i_see_the_current_date_and_time_selected
+    expect(page).to have_selector(
```

Comment:
Minor, but I think we can use `find_field(...).value` here e.g. as in the [image specs](https://github.com/alphagov/content-publisher/blob/master/spec/features/editing_images/choose_lead_image_spec.rb#L39).

Reply:
We use `have_selector` elsewhere in the specs so I'm going to leave it as is.